lib/util/config-types.ts

```diff
@@ -243,6 +243,7 @@ export interface ItineraryCostWeights {
 }
 
 export interface ItineraryConfig {
+  allowUserAlertCollapsing: boolean
```

Comment:
Yes, but make it optional:
```suggestion
  allowUserAlertCollapsing?: boolean
```